pkg/sizing/algorithm_test.go

```diff
@@ -33,3 +33,14 @@ func Test_AlgorithTest_Algorith(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func Test_CoresNodeInvariant(t *testing.T) {
```

Comment:
@trevorwhitney I may misunderstand the logic of the algorithm but it fails here in that it demands more cores than is available on the nodes.

Reply:
That sounds like a bug. It should add additional nodes if there aren't enough cores. After a brief look at the code nothing jumps out, would have to dig in more.